README.md

```diff
@@ -423,7 +423,7 @@ Companies that have allowed us to speak about our work can be found here. Many m
 | [Primitive Hyper](https://primitive.xyz/) | Jan 2023 | 8 |  |  |
 | [Polygon Edge](https://polygon.technology/) | Jan 2023 | 6 |  |  |
 | [Optimism](https://www.optimism.io/) | Dec 2022 | 8 |  |  |
-| [Paxos](https://paxos.com/) | Dec 2022 | 1 |  |  |
+| [Paxos PayPal PYUSD](https://paxos.com/pyusd/) | Dec 2022 | 1 | | [📄✅](reviews/2022-12-paxos-paypal-pyusd-securityreview.pdf)
```

Comment:
I don't think you need it (i.e., I don't think this breaks the table formatting) but you may want to add a vertical pipe `|` at the end of the line to be consistent with the other lines. Otherwise, looks good!